ReSwift/CoreTypes/Store.swift

```diff
@@ -40,11 +40,29 @@ open class Store<State: StateType>: StoreType {
 
     private var isDispatching = false
 
+    /// Indicates if new subscriptions attempt to apply `skipRepeats` 
+    /// by default.
+    let subscriptionsAutomaticallySkipRepeats: Bool
```

Comment:
Good point -- I really wanted to make this `public` but forgot. My rationale: if this is a major change in "modes", it might be useful to query for the currently active mode. 

But I see that since it's configured once and cannot change, you are supposed to know the mode anyway. So `private` makes more sense.